scripts/post-processing/1-run_normalization.R

```diff
@@ -108,6 +99,15 @@ if (any(is.na(match(opt$algorithm, all.algorithms)))) {
 # Read in a tsv file of data
 dataset <- readr::read_tsv(opt$data)
 
+# Find which column has gene info
```

Comment:
It will keep all the columns with "gene" in the column name. Which I think is fine. For line 179 As I had it, it would have by default named the colnames "genes.1" and "genes.2" if more than one column existed, but I just got rid of the naming of ` "genes"` part. Now it will carry over whatever the column names are.